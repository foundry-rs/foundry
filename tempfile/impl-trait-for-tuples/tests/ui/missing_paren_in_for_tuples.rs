trait Test {
    type Test;
}

#[impl_trait_for_tuples::impl_for_tuples(1)]
impl Test for Tuple {
    for_tuples!( type Test = #( Tuple::Test ),*; );
}

fn main() {}
