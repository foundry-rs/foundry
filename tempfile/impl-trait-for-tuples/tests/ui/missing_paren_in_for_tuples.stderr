error: expected parentheses
 --> tests/ui/missing_paren_in_for_tuples.rs:7:30
  |
7 |     for_tuples!( type Test = #( Tuple::Test ),*; );
  |                              ^
